test/e2e/helpers.js

```diff
@@ -460,6 +460,20 @@ const onboardingCompleteWalletCreation = async (driver) => {
   await driver.clickElement('[data-testid="onboarding-complete-done"]');
 };
 
+const onboardingCompleteWalletCreationWithOptout = async (driver) => {
```

Comment:
Nit: Maybe better to treat the "out" as a separate word here (I misread this as `WithOutput` at first :sweat_smile: ) 
```suggestion
const onboardingCompleteWalletCreationWithOptOut = async (driver) => {
```

Reply:
applied in 412d759b7d0132dbc91cbce3748936a22b776994!